components/sections/strategies/SectionStats.tsx

```diff
@@ -39,6 +39,7 @@ const	SectionStats = React.memo(function SectionStats({currentVault, currentStra
 					{title: 'Performance Fee ', details: format.bigNumberAsAmount(currentStrategy.performanceFee, 2, 2, '%')},
 					{title: 'Min Debt Per Harvest ', details: format.bigNumberAsAmount(currentStrategy.minDebtPerHarvest, currentVault.decimals, 4)},
 					{title: 'Max Debt Per Harvest ', details: format.bigNumberAsAmount(currentStrategy.maxDebtPerHarvest, currentVault.decimals, 4)},
+					{title: 'KeepCRV ', details: format.bigNumberAsAmount(currentStrategy.keepCRV, 0)},
```

Comment:
If the strategy doesn't have `keepCRV` field, it should display `N/A`.

![image](https://user-images.githubusercontent.com/46446076/172976614-9e658d77-eec9-4b93-84e1-be9102d37e6d.png)

Reply:
This kinda strategies should not be tracked on `Track` page.

Reply:
Then, what page is good for tracking keepcrv?
I created new page named Track by myself.

I'd like to know your opinion.

Reply:
`Track` page is the correct one where we should track strategies with `keepCRV` field.
What I am saying is that, currently it has some strategies without `keepCRV` field.
Like I mentioned in the above image.

Reply:
Is this none `keepCRV` strategy from track page?

Reply:
[Lido Eth 2.0 Staking](https://etherscan.io/address/0x0967aFe627C732d152e3dFCAdd6f9DBfecDE18c3#readContract) doesn't have `keepCRV` field, but it is on `Track` page.

![image](https://user-images.githubusercontent.com/46446076/173100323-498a655c-3b98-4702-b0de-9d4142517db5.png)